src/views/Home/Home.jsx

```diff
@@ -0,0 +1,80 @@
+import { useState, useEffect } from 'react';
+import React from 'react';
+import {
+  fetchPokedex,
+  fetchRegionList,
+} from '../../services/fetchdata/fetchdata';
+import PokmeonThumb from '../../components/PokemonThumb/PokemonThumb';
+import Controls from '../../components/Controls/Controls';
+import { Link, Route, useHistory, useRouteMatch } from 'react-router-dom';
+import style from './Home.css';
+
+import PokemonDetails from '../PokemonDeatils/PokemonDetails';
+import ItemList from '../ItemList/ItemList';
+import { Switch } from 'react-router-dom';
+import HomeDetail from '../../components/HomeDetail/HomeDetail';
+
+export default function Home() {
+  const [pokidex, setPokedex] = useState([]);
+  const [regionList, setRegionList] = useState('');
+  const [region, setRegion] = useState('kanto');
+  const [loading, setLoading] = useState(true);
+
+  const histroy = useHistory();
+
+  useEffect(() => {
+    const fectData = async () => {
+      const data = await fetchPokedex(region);
+      setPokedex(data);
+      const { results } = await fetchRegionList();
+      setRegionList(results);
+      setLoading(false);
+    };
+    fectData();
+  }, [region]);
+  const handleChange = (e) => {
+    setRegion(e.target.value);
+  };
+
+  const handleClick = () => {
+    histroy.push('/');
+  };
+
+  if (loading) return <p>Loading...</p>;
+
+  return (
+    <>
+      <div className={style.pokedex}>
+        <Controls
+          regionList={regionList}
+          handleChange={handleChange}
+        ></Controls>
+        <div className={style.pokemonThumbList}>
+          {pokidex.map((pokemon) => (
+            <div key={pokemon.entry_number}>
+              <Link
+                onClick={handleClick}
```

Comment:
I don't  know that I would combine an `onClick` and a `Link` like this. I would instead use the `url` from `useRouterMatch` to construct the url on this link